freqtrade/persistence/trade_model.py

```diff
@@ -915,6 +918,12 @@ def select_filled_or_open_orders(self) -> List['Order']:
                 or (o.ft_is_open is True and o.status is not None)
                 ]
 
+    def set_kval(self, key: str, value: Any) -> None:
```

Comment:
i understand what you're trying to say ("set key value" - but shorter) - however i think for a user (no prorammer / developer) this will be confusing (what's a kval, exactly?)

maybe using something a bit more custom - like trade_ (not just for this - but also for the classes / table) - which however makes it clearer what this is (storage for user data tied to trades).

Reply:
Addressed through multiple commits.